backend/ModelLayer/Models/Organization.cs

```diff
@@ -0,0 +1,14 @@
+﻿using ModelLayer.Models.Base;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ModelLayer.Models
+{
+    public class Organization : BaseEntity
+    {
+        public Address Address { get; set; }
+
+        public List<User> Employees { get; set; }
```

Comment:
Ich glaub da sollten wir einen Unterschied machen.
User - Klasse sind die benutzer vom CRM. EMployee muss nicht unbedingt auch beim CRM angemeldet sein oder?
Könntest du das in das Klassenmodell erweitern?

Reply:
ne passt. aber aus deinem klassendiagramm geht doch bereits eine Contact klasse hervor. ist das dann nicht die, welche hier zutragen kommt?weil grundsätzlich interessieren mich doch nur personen von unternehmen welche auch als kontakt angelegt sind, oder?

Reply:
ja das stimmt. Dachte du unterscheidest nochmal weil du ja Employee geschrieben hast. aber dann kannst gern eine Verbindung zwischen contact und organisation machen :)